Common/Code.ts

```diff
@@ -168,6 +168,28 @@ function getFilenameEncode(equation: string, direction: number) {
  */
 function reEncode(equation: string) {
```

Comment:
I saw an update to re-encode that's not in de-encode. Do they derender right?

Reply:
They derender to the correct TeX commands: 
<img width="639" height="90" alt="image" src="https://github.com/user-attachments/assets/82327fa0-b23e-4ba9-ad36-a73e4f33c746" />

I _could_ replace those commands with the unicode versions, but that would probably be unexpected for people who typed the commands in themselves and would probably break for more advanced uses like `\acute{3x^{\tilde{9}}}` (which is completely valid)